packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.types.ts

```diff
@@ -326,7 +326,7 @@ export interface IDetailsListProps extends IBaseProps<IDetailsList>, IWithViewpo
    * Optional override properties to render details header. The definition for IDetailsHeaderProps
    * can be found under the DetailsHeader component.
    */
-  detailsHeaderProps?: IDetailsHeaderProps;
+  detailsHeaderProps?: Partial<IDetailsHeaderProps>;
```

Comment:
@KevinTCoughlin I advised on using Partial in here due to some required props that `IDetailsHeaderProps` is imposing. Would that be an acceptable type on a prop for this purpose? We really shouldn’t drill holes for individual DetailsHeader props and have this one prop that allows passing a prop object. There are a couple props on DetailsList that already drilled them and could be potentially deprecated in favor of this one.